upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -1345,13 +1345,13 @@ func (c *awsCloudImplementation) DefaultInstanceType(cluster *kops.Cluster, ig *
 		// Some regions do not (currently) support the m3 family; the c4 large is the cheapest non-burstable instance
 		// (us-east-2, ca-central-1, eu-west-2, ap-northeast-2).
 		// Also some accounts are no longer supporting m3 in us-east-1 zones
-		candidates = []string{"m3.medium", "c4.large"}
+		candidates = []string{"c5.large", "c4.large", "m3.medium"}
```

Comment:
I wonder if we want to consider a t3.large instead. We're more likely to be memory constrained.

Reply:
t3.* has CPU credits and may be throttled on master startup. Also, it has slower throttled network. Should be ok for a small clusters. If we see issues, I would go with m5.large.

Reply:
I think it's reasonable to tune the defaults for small clusters.

Reply:
t3.* default to unlimited, so won't be CPU throttled on startup. If the CPU utilization is at or below the baseline over the 24 hour period (or lifetime) then there won't be an overage charge. The only issues would seem to be the slower network or the possibility of higher cost if the CPU is used at a high sustained rate.

Reply:
I forgot about the part of Unlimited by default. In this case, it may be a good idea to set t3.medium as default for masters also. It should stay around 10% CPU usage, which is not charged extra. Network throttling is not so bad for a small cluster (baseline is 0,25 Gbit/s).